TACC/Core-CMS
# Task/FP-1550-custom-text-in-admin-ui

## Overview

Suppot custom language strings.

## Related

* [FP-1550](https://jira.tacc.utexas.edu/browse/FP-1550)

## Changes

- Create and document folder for language strings.
- Add setting to use new folder for language strings.

## Testing

1. Create file `taccsite_cms/locale/en/LC_MESSAGES/django.po`.
2. Add this content to the file:

    ```
    #: django-cms
    #: cms/cms_toolbars.py:400
    msgid "Publish page changes"
    msgstr "Dragon approves changes"

    #: djangocms-bootstrap
    #: contrib/bootstrap4_grid/models.py:42
    msgid ""
    "Defines if the grid should use fixed width (<code>.container</code>) or "
    "fluid width (<code>.container-fluid</code>)."
    msgstr ""
    "Defines a grid of water magic with fluid and frozen containers. "
    "No animals will be harmed by the creation of this grid."
    ```

## Screenshots

…
